.env.example

```
@@ -1,6 +1,10 @@
 OPENCAGE_API_KEY=
 SERVER_CRYPTO_KEY=
 SERVER_INIT_VECTOR=
+# Server-held token for btcmap-api's submit_place RPC (origin "btcmap-website").
+# Required in production (Netlify env var); without it /api/submit-place
+# returns 503. Never expose with a VITE_ prefix.
+BTCMAP_IMPORT_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BTCMAP_IMPORT_TOKEN key should go before the OPENCAGE_API_KEY key

(UnorderedKey)